keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/ImageHandler.kt

```diff
@@ -153,7 +153,6 @@ class ImageHandler(
               "package" to artifactRef.substringAfterLast("/"),
               "regions" to artifact.vmOptions.regions,
               "storeType" to artifact.vmOptions.storeType.name.toLowerCase(),
-              "user" to "keel",
```

Comment:
@emjburns Turns out the only difference from this orca task and all the others is that we overriding `user` here (which is set to `"Spinnaker"` by default in the `Job` type alias).

Reply:
Ahhh great :)